r_basics_introduction/r_basics_introduction.md

```diff
@@ -0,0 +1,784 @@
+
+![`The first few lines of the introduction_exercises.Rmd file`](media/introduction_exercises.png)<!-- style = "max-width:700px;" -->
+
+Read through the instructions and complete the assigned tasks:
+
+* import data from a .csv file
+* use the RStudio data viewer to answer some questions
+
+### Solutions
+
+You can open the `introduction_solutions.Rmd` file to see the answers.
+
+To import data, you run the cell indicated.  Click on the image below to show a short clip of what this looks like.
+
+<div style="display:none">@gifPreload</div>
+
+<figure>
+  <img src="https://github.com/arcus/education_modules/blob/r_basics_introduction/r_basics_introduction/media/run_import_chunk.png?raw=true" height="540" width="878" alt="RStudio can create a new project that gets its contents from a git repository." data-alt="https://github.com/arcus/education_modules/blob/r_basics_introduction/r_basics_introduction/media/run_import_chunk.gif?raw=true" style = "border: 1px solid rgb(var(--color-highlight));">
+
+<figcaption style = "font-size: 1em;">Click on the image to play the demo of running the chunk that imports data.</figcaption>
+</figure>
+
+To answer the questions using the file viewer, click on the name of a data frame (like `covid_testing`) to open a view into the data in a new tab.  Click on the image below to show a short clip of what this looks like.
+
+<div style="display:none">@gifPreload</div>
+
+<figure>
+  <img src="https://github.com/arcus/education_modules/blob/r_basics_introduction/r_basics_introduction/media/viewer.png?raw=true" height="540" width="878" alt="RStudio can create a new project that gets its contents from a git repository." data-alt="https://github.com/arcus/education_modules/blob/r_basics_introduction/r_basics_introduction/media/viewer.gif?raw=true" style = "border: 1px solid rgb(var(--color-highlight));">
+
+<figcaption style = "font-size: 1em;">Click on the image to play the demo of opening a viewer of a data frame.</figcaption>
+</figure>
+
+Here are the answers to the questions posed in `introduction_exercises.Rmd`:
+
+1. How many rows are in the data frame? How many columns?
+
+There are 15,524 rows and 17 columns. This information can be seen at the bottom of the Data Viewer, and also in the Environment pane. Note that in R parlance, rows are observations, and columns are variables.
+
+2. Go ahead and try to edit one of the values in this viewer. You will find that you can't. It would have been easy for the RStudio programmers to allow editing of specific values, but they decided not to add that feature. Why do you think this was designed that way?
+
+This would encourage point-and-click workflows that are not reproducible.
+
+3. Each row in *covid_testing* represents a single COVID19 test. The value in the *pan_day* column indicates the day (after the start of the pandemic) on which the test sample was taken. On which "pandemic day" was the first test sample taken? The last?
```

Comment:
weirdness with italics and underscores and things ... perhaps an escape is needed?

Reply:
whoops yep you are right!